### PR TITLE
Replace less with scss.

### PR DESCRIPTION
**Pull request by JoshDunnDev**

### CODE REVIEW

File: test/components/app.scss [hunk 1]
```diff
@@ -2,4 +2,4 @@
  * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
  */
 @import url('https://fonts.googleapis.com/icon?family=Material+Icons');
-@fa-font-path: "@{bedrock-module-path}/@fortawesome/fontawesome-free/webfonts";
+$fa-font-path: "@{bedrock-module-path}/@fortawesome/fontawesome-free/webfonts";
```
Comment (dlongley) [inline test/components/app.scss]: `bedrock-module-path` is a var here that needs to be updated to use the scss syntax.

We also need support for the var in scss via `bedrock-views` or some other method (like always setting it in top-level applications). See: https://github.com/digitalbazaar/bedrock-views/issues/43

@davidlehn, @gannan08, @mattcollier -- do any of you guys have thoughts on what we want to do moving forward here? What we may do to remove the need to specify frontend dependencies could also be impacted (or help solve this issue).

**Review by dlongley — CHANGES_REQUESTED** [review]

File: test/components/app.scss (hunk 1 above)
Comment (JoshDunnDev) [reply test/components/app.scss]: I have updated to use #{$bedrock-module-path} instead. I believe you are correct that we will need to make some adjustments to use $bedrock-module-path in `bedrock-views`.

**Comment by dlongley** [comment]
cc: @davidlehn -- this PR isn't going to work until we have some kind of scss thing for the modules path. We should consider this when updating bedrock-views to help fix other dev issues.

**Comment by mattcollier** [comment]
@JoshDunnDev I believe at this time you can use bedrock-fontawesome to address the issues with this PR.

**Comment by JoshDunnDev** [comment]
Closing due to being outdated and is no longer an issue.

**Pull request closed without merging**